contrib/codegen-tools/codegen/src/main/ops/org/nd4j/codegen/ops/SDBaseOps.kt

```diff
@@ -480,6 +481,38 @@ fun SDBaseOps() =  Namespace("BaseOps"){
         }
     }
 
+
+    Op("sparseToDense") {
+        javaPackage = "org.nd4j.linalg.api.ops.compat"
+        javaOpClass = "CompatSparseToDense"
```

Comment:
@treo sparse to dense is also a new op. Used in model import and works fine, just never fully mapped or exposed.